composer.json

```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php" : "^7.0",
-        "illuminate/support": "~5.3.0|~5.4.0|~5.5.0"
+        "illuminate/support": "~5.3.0|~5.4.0|~5.5.0|~5.6.*"
```

Comment:
I think this needs to be "5.6.*" or "~5.6.0" or ">5.6" right?

Reply:
ok ill update it and send another merge request